svelte-cloudinary/src/lib/helpers/getCldImageUrl.ts

```diff
@@ -1,4 +1,5 @@
 import { constructCloudinaryUrl } from '@cloudinary-util/url-loader';
+import { checkCloudinaryCloudName } from '../cloudinary.ts';
```

Comment:
```suggestion
import { checkCloudinaryCloudName } from '../cloudinary.js';
```

Reply:
I have made the changes :+1:  
I am not used to with typescript, so  can you please explain why we use ```.js``` for a ```typescript``` file ?

Reply:
I think this will be a good explanation https://x.com/mattpocockuk/status/1711697978140852695?s=20